src/communication/network_interface_sr.c

```diff
@@ -7564,6 +7579,10 @@ smmon_get_tran_info (THREAD_ENTRY * thread_p, unsigned int rid, char *request, i
   // Size of transaction id (OR_INT_SIZE)
   // and current memory usage (OR_INT64_SIZE)
   size += ((OR_INT_SIZE + OR_INT64_SIZE) * info.num_tran);
+  if (size != 0)
```

Comment:
```suggestion
  if (size == 0)
```
Isn't it for the case where there is no trx?

Reply:
I miss it. Done.